### PR TITLE
Upgrade to the last state of Laravel 9.2 with Vite

### PR DESCRIPTION
**Pull request by antonioribeiro**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,13 +2,31 @@
...
     "scripts": {
         "dev": "vite",
-        "build": "vite build"
+        "build": "vite build",
+        "development": "mix",
+        "watch": "mix watch",
+        "watch-poll": "mix watch -- --watch-options-poll=1000",
+        "hot": "mix watch --hot",
+        "prod": "npm run production",
+        "production": "mix --production",
```
Comment (luislavena) [inline package.json]: Are the `mix` tasks correct? Thought switching to Vite will remove those.

**Review by luislavena — CHANGES_REQUESTED** [review]
Perhaps this is worth rebase from `main` as the first commit to solve conflicts shows both Vite and mix listed?

Thank you!

File: package.json (hunk 1 above)
Comment (antonioribeiro) [reply package.json]: Fixed!

**Comment by luislavena** [comment]
@antonioribeiro rebased, combined and merged in 72b915124cdb18c23344fad6c3a382931269022d.

Thank you!

**Pull request closed without merging**